UKHO.FmEssFssMock.API/UKHO.FmEssFssMock.API/Helpers/EnumHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel;
+using System.Reflection;
+
+namespace UKHO.FmEssFssMock.API.Helpers
+{
+    public static class EnumHelper
+    {
+        public static string GetEnumDescription(Enum enumVal)
+        {
+            System.Reflection.MemberInfo[] memInfo = enumVal.GetType().GetMember(enumVal.ToString());
```

Comment:
Could we please give readable variable name

Reply:
Updated